src/components/Plans/Checkout/PaymentMethod/PaymentMethod.js

```diff
@@ -143,7 +144,7 @@ const PaymentType = ({ paymentMethodType, optionView, paymentMethod }) => {
           case paymentType.transfer:
             return <Transfer optionView={optionView} paymentMethod={paymentMethod}></Transfer>;
           default:
-            return null;
+            return <MercadoPagoArgentina optionView={optionView} />;
```

Comment:
For me the default should be ```<CreditCard optionView={optionView}></CreditCard>``` because Mercadopago is available anly for Argentina and for the Mercadopago would add:
```
case paymentType.mercadoPago:
            return <MercadoPagoArgentina optionView={optionView} />;
```

What do you think?

Reply:
Por defecto Se devuelve `CreditCard` ...